src/makeCalculator.js

```diff
@@ -37,7 +37,48 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
+    digit: 0,
+
+    add: (val) => {
+      calc.digit += val;
+
+      return this;
+    },
+    subtract: (val) => {
+      calc.digit -= val;
+
+      return this;
+    },
+    multiply: (val) => {
```

Comment:
```suggestion
    add: (val) => {
      calc.digit += val;

      return this;
    },
    
    subtract: (val) => {
      calc.digit -= val;

      return this;
    },
    
    multiply: (val) => {
```

Next time pls add empty line between methods